views/includes/check-your-answers/trusts.html

```diff
@@ -0,0 +1,34 @@
+<h3 class="govuk-heading-m">Trust {{ trust_count }}</h3>
```

Comment:
instead of trust_count and counting them, i think you could appData.trusts.length here
appData.trusts.length if appData.trusts else '0'

Reply:
Didn't realise the count was needed on page, this is fine